src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -618,4 +619,10 @@ private Link handleUnknownLink(de.tudarmstadt.ukp.wikipedia.parser.Link link)
 		}
 	}
 
+	private boolean isImage(de.tudarmstadt.ukp.wikipedia.parser.Link link)
```

Comment:
I thought we would call this `validateLink` or something like that. It would be nice to check for the presence of files and other things too. As far as I understand, we would remove stuff from the namespaces you added below, but they are not involved in the validation function. Is that supposed to be like this ?

Reply:
Also, might be good to add a test for this.

Reply:
So the extra namespaces should be taken care of by the existing code, because they have a namespace like "File: somefile" but the images were not being recognised because they did not have a namespace,  they were simple strings (not even a [[ ]]) but they were inside the <galery> </galery> tags. So thats why the special function.

Reply:
Can we have a comment to this effect in the file too please?